queries.sql

```diff
@@ -8,4 +8,71 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5; 
 SELECT * FROM animals WHERE neutered = TRUE; 
 SELECT * FROM animals WHERE name <> 'Gabumon'; 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3; 
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3; 
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Take%20a%20screenshot%20of%20the,the%20results%20of%20your%20actions.) 

> you are to delete all records in the animals table, then roll back the transaction.
> After the rollback verify if all records in the animals table still exists. below is an example 👇  
```
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

Reply:
@Davidosky007 I Have also done this and uploaded the screenshot!